src/router/routes.js

```diff
@@ -242,6 +235,14 @@ module.exports = [
 			unbounce: true,
 		},
 	},
+	{
+		path: '/lend/:id(\\d+)',
+		redirect: '/lend-beta/:id'
```

Comment:
Won't this still have pages loading at the `/lend-beta/###` url? Why not define the BorrowerProfile page component here as well and just use `/lend/###`...

Reply:
Yeah.. sounds right. @emuvente would this imply setting a `canonical url` for the `Borrower Profile` page?